src/Expander/Error.hs

```diff
@@ -140,11 +140,11 @@ instance Pretty VarInfo ExpansionErr where
     vsep [ pp env stx <> text ":"
          , group $ vsep [ group $ hang 2 $
                           vsep [ text "Used in a position expecting"
-                               , pp env expected
+                               , pp env actual
```

Comment:
It looks to me like the variable names are also wrong here...

Reply:
fixed, thanks